numpy/core/src/multiarray/common.c

```diff
@@ -484,7 +484,7 @@ check_has_rank_0(PyArrayObject *v)
     }
     /* Remove this if when the deprecation expires */
     if (PyArray_SIZE(v) == 1) {
-        /* Numpy 1.19.0, 2019-12-18 */
+        /* Numpy 1.21.3, 2021-08-17 */
```

Comment:
Not to be too picky, but I presume this is not going to be backported, so should probably say `1.22.0`.

Reply:
Ah yes. Should the deprecation note go in `doc/source/release/1.22.0-notes.rst`, too? (That file is still more or less empty.)

Reply:
It should go as a deprecation into `doc/release/upcoming_changes`, check the `README` in that file.  The release note is incorrect as is.